### ISSUE TITLE
Hass io installation

### ISSUE THREAD

**Issue by ziat007** [rytilahti/python-yeelightbt#12]
Could a hass io repository be added for simple (no root access required) installation?

Without this I don't think it is easy to install this component. (Hass io only grants access to a virtual environment with home assistant preinstalled)

**Comment by bachulator** [comment]
Yes integration with hassio would be super cool. We need a plug-in for this extension I assume.

**Comment by rytilahti** [comment]
Pull requests for adding such are welcome, as I don't use hassio myself I will not be working on this.

**Comment by fernandex00** [comment]
Some news about this?

**Comment by LorenzBischof** [comment]
Wouldn't the `custom deps deployment` addon work?

**Comment by vicvancooten** [comment]
I know this is old, but you can install it in hass.io using the following config for `custom deps deployment`.

```
{
  "pypi": [
    "git+https://github.com/rytilahti/python-yeelightbt/",
    "construct==2.9.25"
  ],
  "apk": []
}
```

**Comment by matteos1** [comment]
> I know this is old, but you can install it in hass.io using the following config for `custom deps deployment`.
> 
> ```
> {
>   "pypi": [
>     "git+https://github.com/rytilahti/python-yeelightbt/",
>     "construct==2.9.25"
>   ],
>   "apk": []
> }
> ```

could you explain how is possible install in hassio?

**Comment by vicvancooten** [comment]
> > I know this is old, but you can install it in hass.io using the following config for `custom deps deployment`.
> > ```
> > {
> >   "pypi": [
> >     "git+https://github.com/rytilahti/python-yeelightbt/",
> >     "construct==2.9.25"
> >   ],
> >   "apk": []
> > }
> > ```
> 
> could you explain how is possible install in hassio?

What exactly is unclear? In hass.io you have to install the `custom deps deployment` add-on. You can use that as the configuration for the add-on. Run it once and you should be done.

**Comment by lasry1** [comment]
I've tried to install it through the custom deps deployment add-on and it doesn't work. I have an error message saying:


Got exception: cannot import name 'SymmetricMapping'

**Comment by rytilahti** [comment]
The problem with symmetricmapping should be fixed by #28, the custom component may or may not work + it will also capture the device for the runtime of homeassistant (which can be problematic if your bluetooth adapter does not support multiple connections, or if you want to control it outside homeassistant) and will thus also drain the battery (in case of candelas).

If someone is interested in working to make it more usable, I can give some help if needed, but I'm not personally going to put much effort on this project.

**Comment by rytilahti** [comment]
Closing as stale.
